langwatch/src/components/evaluations/wizard/EvaluationWizard.tsx

```diff
@@ -374,6 +397,13 @@ const WizardSidebar = memo(function WizardSidebar({
                   Show Code
                 </Button>
               )}
+            {step === "results" && !executionMethod?.startsWith("realtime") && (
```

Comment:
maybe instead of execution_method startsWith we should check the `task` if it's realtime?